doc/howto_release.md

```diff
@@ -323,17 +327,17 @@ fetch the file from the release which was generated by a workflow
 linked to the tag:
 
 ```bash
-wget https://github.com/OSGeo/grass/releases/download/${VERSION}/ChangeLog.gz \
-    -O ChangeLog_${VERSION}.gz
+wget https://github.com/OSGeo/grass/releases/download/$REL_VERSION/ChangeLog.gz \
+    -O ChangeLog_$REL_VERSION.gz
 ```
 
 ### Get the source code tarball
 
 Fetch a tarball from GitHub we also publish on OSGeo servers:
 
 ```bash
-wget https://github.com/OSGeo/grass/archive/${VERSION}.tar.gz -O grass-${VERSION}.tar.gz
-md5sum grass-${VERSION}.tar.gz > grass-${VERSION}.md5sum
+wget https://github.com/OSGeo/grass/archive/$REL_VERSION.tar.gz -O grass-$REL_VERSION.tar.gz
+md5sum grass-$REL_VERSION.tar.gz > grass-$REL_VERSION.md5sum
```

Comment:
I suppose this step was skipped, the tarballs at https://download.osgeo.org/grass/grass84/source/grass-8.4.1.tar.gz and https://github.com/OSGeo/grass/releases/download/8.4.1/grass-8.4.1.tar.gz deviates, insofar the generated `include/VERSION_GIT` is missing from the former.

The file is generated by:

https://github.com/OSGeo/grass/blob/8511d5b4f3c98cc359c0c7729a8b28be169ccbe1/.github/workflows/create_release_draft.yml#L51-L52

Maybe this step should be added to the root_release.md file?

Reply:
Now I see, the `wget https://github.com/OSGeo/grass/archive/$REL_VERSION.tar.gz` should be updated to:
`wget https://github.com/OSGeo/grass/releases/download/8.4.1/$REL_VERSION.tar.gz`

Reply:
The `...archive/$REL_VERSION.tar.gz` file is a straightforward tarball of repo, not the one with generated files.

Reply:
Today:

```
echo $REL_VERSION
8.4.1
```

Since I want to avoid hardcoded version, like this?

`wget https://github.com/OSGeo/grass/releases/download/$REL_VERSION/$REL_VERSION.tar.gz`

hence, in the text:

`wget https://github.com/OSGeo/grass/releases/download/$REL_VERSION/$REL_VERSION.tar.gz -O grass-$REL_VERSION.tar.gz`

?

Unfortunately I get 404...

Reply:
@nilason please check the related fix in 9e4a6afcf5be3ef55c417ffd7b7e4ddc284fc737

Reply:
I'll re-update the tarballs on the grass/download servers.

Reply:
Tarballs updated on the servers. Thanks for catching this, @nilason!

Reply:
Great!